applications/psd_pipeline/metadata.json

```diff
@@ -20,6 +20,7 @@
         "changelog": {
             "1.0": "Initial Release"
         },
+        "dockerfile": "applications/psd_pipeline/Dockerfile",
```

Comment:
Seems like the Dockerfile should be picked up automatically as being in the same folder as the app, could we confirm with `./holohub build-container psd_pipeline --dryrun` whether this change is required?